packages/hash/frontend/sentry.client.config.ts

```diff
@@ -2,19 +2,36 @@
 // https://docs.sentry.io/platforms/javascript/guides/nextjs/
 
 import * as Sentry from "@sentry/nextjs";
+import { Replay } from "@sentry/replay";
+import {
+  SENTRY_DSN,
+  SENTRY_REPLAYS_SAMPLING_RATE,
+  VERCEL_ENV,
+} from "./src/lib/public-env";
 
-const dsn = process.env.NEXT_PUBLIC_SENTRY_DSN;
+const replaysSamplingRate = Number.parseInt(
+  SENTRY_REPLAYS_SAMPLING_RATE ?? "0",
+  10,
+);
```

Comment:
You might want to use `Number.parseFloat` here instead of `Number.parseInt`. 

The allowed range is `0.0 <= replaysSamplingRate <= 1.0`. Parsing to int means you'll endup with either `0` or `1`, no values in between.

Reply:
Many thanks for catching this typo @ryan953! I replaced `parseInt` with `parseFloat` 👍